### PR TITLE
Added neurogliaform morphology

### LINKED ISSUE #367: NTR: neurogliaform morphology
**Term:** 
neurogliaform morphology

**Definition:**
A cell morphology that inheres in neurons which have a small round soma, a large number of short, smooth, or slightly beaded primary dendrites that give rise to only a few secondary branches, and a branched axon that establishes a dense axonal mesh with thin shafts. 
PMID:17122314

**Parent Class:**
Cell Morphology #364

**Others:**
ilxtr:NeurogliaformPhenotype

### PR DESCRIPTION
**Pull request by shawntanzk**
Fixes #367

### CODE REVIEW

**Review by patrick-lloyd-ray — COMMENTED** [review]
Looks good to me.

**Review by dosumis — DISMISSED** [review]

**Comment by shawntanzk** [comment]
@dosumis sorry about that, was fixing some clashes and somehow ended up dismissing review so need a re-approval

**Review by patrick-lloyd-ray — APPROVED** [review]

**Pull request merged**
